modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -112,14 +108,14 @@ System Info:                 <9>
  OS Image:                           OpenShift Enterprise
  Operating System:                   linux
  Architecture:                       amd64
- Container Runtime Version:          docker://1.13.1
+ Container Runtime Version:          1.13.1
```

Comment:
this will be something like `cri-o://1.12.5-6.rhaos4.0.git80d1487.el7` reflecting the crio RPM included in RHCOS